README.md

```diff
@@ -332,6 +332,10 @@ public class DeepLinkActivity extends Activity {
 
 if you use configurable path segments
 
+### Upgrading
+
+When upgrading to the latest version you might need to make some changes. Read about it [here](UPGRADING.md).
```

Comment:
Can we specify which versions to consider?
```suggestion
When upgrading to 5.x+ you may experience some breaking API changes. Read about them [here](UPGRADING.md).
```

Or did you want to keep it more general and specify version considerations in Upgrading.md?